src/github.com/travis-ci/jupiter-brain/vsphere.go

```diff
@@ -301,10 +302,27 @@ func (i *vSphereInstanceManager) createClient(ctx context.Context) (*govmomi.Cli
 	client.Client.RoundTripper = &soapBreakerRoundTripper{
 		rt: client.Client.RoundTripper,
 		cb: gobreaker.NewCircuitBreaker(gobreaker.Settings{
-			Name: "vSphere govmomi",
+			Name:     "vSphere govmomi",
+			Interval: 10 * time.Minute,
 			ReadyToTrip: func(counts gobreaker.Counts) bool {
 				failureRatio := float64(counts.TotalFailures) / float64(counts.Requests)
-				return counts.Requests >= 3 && failureRatio >= 0.6
+				return counts.Requests >= 5 && failureRatio >= 0.6
+			},
+			OnStateChange: func(name string, from gobreaker.State, to gobreaker.State) {
+				stateToString := func(state gobreaker.State) string {
+					switch state {
+					case gobreaker.StateClosed:
+						return "closed"
+					case gobreaker.StateHalfOpen:
+						return "half-open"
+					case gobreaker.StateOpen:
+						return "open"
+					default:
+						return "unknown"
+					}
+				}
+
+				i.log.WithField("state_from", stateToString(from)).WithField("state_to", stateToString(to)).Info("circuit breaker state changed")
```

Comment:
I think it's more important to get the underlying error to Sentry, maybe? Which also reminds me, this error should probably send a different HTTP status code back?